leases/enums.py

```diff
@@ -9,3 +9,4 @@ class LeaseStatus(TextChoices):
     EXPIRED = "expired", _("Expired")
     ERROR = "error", _("Error")
     PAID = "paid", _("Paid")
+    TERMINATED = "terminated", _("Terminated")
```

Comment:
does Terminated need translation in ./locale/{fi,sv}/LC_MESSAGES/django.po?

Reply:
It's not critical since we don't rely on it for the translations of the emails or anything, but it would be nice to have.

However, we do need translations for the [notice email subject](https://github.com/City-of-Helsinki/berth-reservations/blob/315c0e8566e430c0b14c2c2aa66a077d4f6fbd92/leases/notifications/types.py#L10-L13)